### PR TITLE
Update build config to minimize bundle sizes.

### PR DESCRIPTION
**Pull request by Hyperkid123**
Build sizes should be about 60% smaller. It would be best if packages would slowly migrate to absolute import paths so we can remove the babel transform import plugin.

The build optimization comes mostly from PF. We were discussing some import changes with PF team for a long time and managed to convince them to let us implement some changes. This should result in major bundle size improvements if you use the latest versions of  PF packages.

### CODE REVIEW

File: packages/inventory-general-info/.gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /index.js
 /index.css
+/TextInputModal.css
```
Comment (karelhala) [inline packages/inventory-general-info/.gitignore]: Can you add these lines to `.npmignore` as well please?
```
!/TextInputModal.css
!/TextInputModal.js
```

File: packages/remediations/src/RemediationButton.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
```
Comment (karelhala) [inline packages/remediations/src/RemediationButton.js]: Can you please add these lines to `.npmignore` as well
```
!/RemediationButton.css
```

File: packages/sources/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@redhat-cloud-services/frontend-components-sources",
-    "version": "0.0.34",
+    "version": "0.0.34-RC3",
```
Comment (karelhala) [inline packages/sources/package.json]: No need to include the `RC` version.

File: packages/utils/.gitignore [hunk 4]
```diff
@@ -1,2 +1,3 @@
 files/
 Styles.css
+helpers.css
```
Comment (karelhala) [inline packages/utils/.gitignore]: Can you please add this to `.npmignore` as well, so we actually ship it.
```
!helpers.css
```

**Comment by karelhala** [comment]
release major

**Comment by karelhala** [comment]
releasemajor

**Comment by karelhala** [comment]
release major

**Comment by AllenBW** [comment]
nacho-bot, throwing a big celebration for the major release there...

**Pull request merged**